Add delay for switch on/off on Fibaro double switch

ctowns opened the pull request:
https://smartthings.atlassian.net/browse/HCS-4161

greens commented on drivers/SmartThings/zwave-switch/src/fibaro-double-switch/init.lua:
```diff
@@ -32,6 +32,9 @@ local ENDPOINTS = {
 }
 
+-- This value is carried over from the DTH
+local ON_OFF_DELAY_SEC = 5
```
you could use `zwave.constants.MIN_DIMMING_GET_STATUS_DELAY`

ctowns replied:
Done

greens commented on drivers/SmartThings/zwave-switch/src/fibaro-double-switch/init.lua:
```diff
@@ -126,6 +129,19 @@ local function switch_report(driver, device, cmd)
+      device:send_to_component(SwitchBinary:Get({}), component)
+    end
+
+    device:send_to_component(SwitchBinary:Set({ target_value = value, duration = 0}), cmd.component)
```
DTH uses `Basic` for the Set, and Binary for the `Get`.

ctowns replied:
Switch to using `Basic`

ctowns commented:
Camille tested this and it is now working with these changes!

Also, I updated the on/off command to use `Basic:Set` as you said and I adjusted the test cases accordingly
